index.html

```diff
@@ -59,8 +59,32 @@ <h1>DuckDuckGo</h1>
 <section class="main-section">
     <!--oguz until id= article-3-->
     <article id="article-1">
-        
-        
+          <div class="privacy">
+            <div class="container">
+              <h1>Privacy, simplified.</h1>
+              <ol>
+                <li>
+                  <span>&#10004</span> 
+                  We don't store your personal info.
+                </li>
+                <li>
+                  <span>&#10004</span> 
+                  We don't follow you around with ads.
+                </li>
+                <li>
+                  <span>&#10004</span> 
+                  We don't track you. Ever.
+                </li>
+              </ol>
+              <a class="home_link" href="https://duckduckgo.com/install?t=hr">Add DuckDuckGo to IE</a>
+              <div>
+                <p>Trusted by <b>millons</b> worldwide</p>
+              </div>
+            </div>
+            <div class="card-img">
+              <img class="img-fluid" src="/img/background.png" alt="main article images" width="1054" height="">
```

Comment:
@oguzkarademir  could you changed path as `src="./img/background.png"` with your path doesn't show up on GitHub